robot/Cumulus/tests/browser/recurring_donations/zedit_day_of_month_verify_next_opportunity.robot

```diff
@@ -0,0 +1,95 @@
+       Go To Page                              Details                        Opportunity                     object_id=${opportunityid}
+       Wait Until Loading Is Complete
+       Click Link                              link=Edit
+       Wait until Modal Is Open
+       Select Value From Dropdown              Stage                          ${stage}
+       Click Modal Button                      Save
+
+
+*** Test Cases ***
+
+Edit An Enhanced Recurring donation record of type open
+    [Documentation]               After creating an open recurring donation using API, The test ensures that the record
+     ...                          can be edited from the ui. A status of closed and the reason for closure can be specified
+     ...                          Verifies the opportunity status reflects the right status after closing
+
+
+    [tags]                                 unstable               W-040346            feature:RD2
+
+    Go To Page                              Details
+    ...                                     npe03__Recurring_Donation__c
+    ...                                     object_id=${data}[contact_rd][Id]
+    Wait Until Loading Is Complete
+    #Validate the number of opportunities on UI, Verify Opportinity got created in the backend
+    Validate Related Record Count           Opportunities                                                  1
+    @{opportunities} =                      API Query Opportunity For Recurring Donation                   ${data}[contact_rd][Id]
+    Edit Opportunity Stage                  ${opportunities}[0][Id]                                        Closed Won
+    Go To Page                              Details
+    ...                                     npe03__Recurring_Donation__c
+    ...                                     object_id=${data}[contact_rd][Id]
+    Edit Recurring Donation Status
+    ...                                     Recurring Period=Monthly
+    ...                                     Day of Month=1
+    Go To Page                              Details
+    ...                                     npe03__Recurring_Donation__c
+    ...                                     object_id=${data}[contact_rd][Id]
+    Wait Until Loading Is Complete
+    ${next_payment_date}                    get next payment date number                                   1
+
+    #Validate that the number of opportunities now show as 2 . Verify the details on the respective opportunities
+    Validate Related Record Count           Opportunities                                                  2
+    @{opportunity} =                        API Query Opportunity For Recurring Donation                  ${data}[contact_rd][Id]
+    Validate Opportunity Details            ${opportunity}[0][Id]        Closed Won                       ${CURRENT_DATE}
+    Validate Opportunity Details            ${opportunity}[1][Id]        Pledged                          ${next_payment_date}
+
+
+
+
```

Comment:
Can you remove this blank lines ? Lint will throw a `TrailingBlankLines` error. Also, run the robot_lint task and remove errors, if any.

Reply:
Thanks, will run the lint task and make the required changes